### PR TITLE
Ref#43697[Admin] manage order

### PR DESCRIPTION
**Pull request by hoangtrungthong**
## Related Tickets:
- [#43697](https://edu-redmine.sun-asterisk.vn/issues/43697)

## WHAT this PR does?
manage order

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by hoangtrungthong** [comment]
ready

**Comment by ghost** [comment]
CI fail nè

**Comment by ghost** [comment]
commented

**Comment by hoangtrungthong** [comment]
ready

File: resources/views/admin/orders/details.blade.php [hunk 1]
```diff
@@ -0,0 +1,102 @@
+<x-app-layout>
+    <x-slot name="slot">
+        <div class="flex items-center justify-center min-h-screen bg-gray-100">
+            <div class="w-full h-full bg-white">
+                <div class="flex justify-between p-4">
+                    <div>
+                        <h6 class="font-bold">Order Date : <span class="text-sm font-medium"> {{ $orderDetails[0]->order->created_at }}</span></h6>
+                        <h6 class="font-bold">Order ID : <span class="text-sm font-medium">{{ $orderDetails[0]->order->id }}</span></h6>
```
Comment (ghost) [inline resources/views/admin/orders/details.blade.php]: localization

File: resources/views/admin/orders/details.blade.php [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+                                    <td class="px-6 py-4">
+                                        {{ $item->price }}
+                                    </td>
+                                    <td class="px-6 py-4">
+                                        {{ $item->price * $item->quantity }}
+                                    </td>
+                                </tr>
+                                @endforeach
+                                @foreach ($orderDetails as $vl)
+                                    <?php $total += $vl['price'] * $vl['quantity']; ?>
```
Comment (ghost) [inline resources/views/admin/orders/details.blade.php]:
```suggestion
                                    @php $total += $vl['price'] * $vl['quantity']; @endphp
```

**Comment by hoangtrungthong** [comment]
ready

**Review by ghost — APPROVED** [review]

**Pull request merged**